pkg/controllers/vscr/podpolling_reconciler.go

```diff
@@ -91,8 +92,13 @@ func (p *PodPollingReconciler) checkScrutinizeContainerStatus(ctx context.Contex
 	}
 	p.VRec.Eventf(p.Vscr, corev1.EventTypeNormal, events.VclusterOpsScrutinizeSucceeded,
 		"Successfully completed scrutinize run for the VerticaDB named '%s'", p.Vscr.Spec.VerticaDBName)
+	stat := &v1beta1.VerticaScrutinizeStatus{}
+	stat.PodName = p.Vscr.Status.PodName
+	stat.PodUID = p.Vscr.Status.PodUID
+	stat.TarballName = pod.Annotations[vmeta.ScrutinizeTarballName]
```

Comment:
Do we need the annotation? Can we just walk the pods arguments and find the tarball name that was passed in.

Reply:
I thought about it. I just thought that the annotation was a cheap way to get the tarballName, but you are right looking up the arguments is simple.